ci/cloudbuild/builds/checkers.sh

```diff
@@ -69,6 +69,13 @@ git_files() {
   fi
 }
 
+CI_CHANGES="$(git_files -z -- "ci/generate-markdown/")"
```

Comment:
If you are going to do dependency management, then use a tool for dependency management.  We have at least two.

doc/packaging.md also depends on the `ci/**/demo-*.Dockerfile` files.

Reply:
> `doc/packaging.md` also depends on the `ci/**/demo-*.Dockerfile` files.

Argh, you're right. I was afraid of this sort of dependency blow up.

My gut says that `FAST_BUT_SOMETIMES_WRONG_CHECKERS` is still a value gain. But I will think it over some more....

Reply:
Maybe what we should have is a `quick-format` script.  Refactor the formatting functions to a common library in ci/cloudbuild/builds/lib, then use the functions from `checkers.sh`, and create a `quick-format.sh` script that only formats what you want.

Reply:
> Refactor the formatting functions to a common library

Oooh, I really like this idea. It would make it way easier to run the formatters after `generate-libraries` (#5821), which I especially want to do as part of (#10237). https://github.com/dbolduc/google-cloud-cpp/blob/4af53dd5eb2a73529788b0997097b14ed105279b/ci/cloudbuild/builds/generate-libraries.sh#L70-L103

(Not working on this now. But maybe I will do it on some rainy day when I am operator)